scripts/install.js

```diff
@@ -45,13 +47,9 @@ function applyProxy(options, cb) {
     var proxyUrl;
 
     if (!er) {
-      ['https-proxy', 'proxy', 'http-proxy'].some(function(setting) {
-        var npmProxyUrl = conf.get(setting);
-        if (npmProxyUrl) {
-          proxyUrl = npmProxyUrl;
-          return true;
-        }
-      });
+      proxyUrl = ['https-proxy', 'proxy', 'http-proxy'].filter(function(setting) {
```

Comment:
I use lodash for that https://lodash.com/docs#find

Surprising node-sass is one of the few codebases without lodash or underscore haha